oauth2/src/main/java/com/okta/spring/boot/oauth/WebClientUtil.java

```diff
@@ -23,7 +23,7 @@
 
 final class WebClientUtil {
 
-    private static final String USER_AGENT_VALUE = ApplicationInfo.get().entrySet().stream()
+    static final String USER_AGENT_VALUE = ApplicationInfo.get().entrySet().stream()
```

Comment:
nit: looks like this can still be private?
```suggestion
    private static final String USER_AGENT_VALUE = ApplicationInfo.get().entrySet().stream()
```